src/SelectInput.tsx

```diff
@@ -98,7 +111,7 @@ function SelectInput<V>({
 	useInput(
 		useCallback(
 			(input, key) => {
-				if (input === 'k' || key.upArrow) {
+				if (input === 'k' || key.upArrow || key.leftArrow) {
```

Comment:
You should determine which keys to listen to based on the list direction. Otherwise if direction is `column`, pressing left arrow would change the selected item, which would be an incorrect behavior.